doc-botonic/docs/plugins/createname-plugin1.md

```diff
@@ -3,6 +3,12 @@ id: createname-plugin1
 title: Prepare the Plugin
 ---
 
+Sometimes you may need a specific feature according to your needs and build your own plugin that doesn't exist yet. In order to achieve this:
+
+- You can create your own plugins to be used locally in your project or as a npm package.
```

Comment:
- You can create your own plugin with an independent node.js package
- Each plugin will be a class which needs to implement the two methods described in this guide.